src/FourBitTwoDisclosureDeviceUnlocker.java

```diff
@@ -109,12 +136,13 @@ private static boolean doSpin(final int numOfSpins) {
      */
     private static boolean doSpin() {
         boolean result;
-        if(state != STATE_NOT_CREATED) {
-            result = dev.spin();
-        } else {
+        if (state == STATE_NOT_CREATED) {
```

Comment:
CheckStyle again suggested we check for equality rather than inequality. I think it is more readable and easier to understand this way.